docs/core/entity.md

```diff
@@ -57,10 +57,16 @@ Whenever you receive a new state from your subscription, you can tell Home Assis
 
 ## Generic properties
 
-The entity base class has a few properties that are common among all entities in Home Assistant. These can be added to any entity regardless of the type. All these properties are optional and don't need to be implemented.
+The entity base class has a few properties common among all Home Assistant entities. These properties can be added to any entity regardless of the type. All these properties are optional and don't need to be implemented.
+
+These properties always called when the state is written to the state machine.
 
 :::tip
 Properties should always only return information from memory and not do I/O (like network requests). Implement `update()` or `async_update()` to fetch data.
+
+Because these properties are always called when the state is written to the state machine, it is important to do as little work as possible in the property.
+
+To avoid calculations in a property method, set the corresponding [entity class or instance attribute](#entity-class-or-instance-attributes) or if the values never change, use [entity descriptions](#entity-description).
```

Comment:
```suggestion
To avoid calculations in a property method, set the corresponding [entity class or instance attribute](#entity-class-or-instance-attributes), or if the values never change, use [entity descriptions](#entity-description).
```